File: widget/src/main/java/id/mochadwi/cataloguewidget/services/StackWidgetService.java
package io.mochadwi.cataloguewidget.services;

import android.content.Intent;
import android.widget.RemoteViewsService;

import io.mochadwi.cataloguewidget.utils.StackRemoteViewsFactory;

public class StackWidgetService extends RemoteViewsService {
    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new StackRemoteViewsFactory(this, intent);
    }
}

File: widget/src/main/java/io/mochadwi/cataloguewidget/services/StackWidgetService.kt
package io.mochadwi.cataloguewidget.services

import android.content.Intent
import android.widget.RemoteViewsService

import io.mochadwi.cataloguewidget.utils.StackRemoteViewsFactory

class StackWidgetService : RemoteViewsService() {
    override fun onGetViewFactory(intent: Intent): RemoteViewsService.RemoteViewsFactory {
        return StackRemoteViewsFactory(this, intent)
    }
}
